pkg/frequencyplans/cflist.go

```diff
@@ -77,6 +77,9 @@ fpChannelsList:
 			}
 		}
 		cfList.Freq = append(cfList.Freq, uint32(fpChannel.Frequency/100))
+		if len(cfList.Freq) > 5 {
```

Comment:
Please see my previous comment.

This is _after_ adding the fifth, we should break when we're _at_ 5.